### PR TITLE
Permissions and ownership of vessel.json

### PR DESCRIPTION
**Pull request by joabakk**
Allows user pi to edit vessel settings file

### CODE REVIEW

File: rpi-setup.sh [hunk 1]
```diff
@@ -109,6 +109,10 @@ cat > $vesselJson <<jsonfile
 }
 jsonfile
 
+sudo chmod 644 $vesselJson
+sudo chown pi $vesselJson
```
Comment (tkurki) [inline rpi-setup.sh]: Could the user be <current user> and not hardcoded `pi`?

Comment (joabakk) [reply rpi-setup.sh]: Yes, but as the script is run as root, wouldn't that become root?

Comment (tkurki) [reply rpi-setup.sh]:
```
tmp tjk$ sudo chown root foo.txt
tmp tjk$ ls -l foo.txt 
-rw-r--r--  1 root  wheel  1 Dec  1 10:49 foo.txt
tmp tjk$ sudo chown $USER foo.txt
tmp tjk$ ls -l foo.txt 
-rw-r--r--  1 tjk  wheel  1 Dec  1 10:49 foo.txt
```
As I understand it $USER is evaluated before calling sudo.
Another method would be to use a variable: assign it, then use it on the sudo line.

Comment (joabakk) [reply rpi-setup.sh]: $SUDO_USER does the trick, it finds the user who invoked sudo. Not sure about group though

Comment (joabakk) [reply rpi-setup.sh]: Group fixed as well

**Comment by joabakk** [comment]
user and group not hard coded any more. Ok to merge

**Pull request merged**